builtin/mainmenu/dlg_config_world.lua

```diff
@@ -69,59 +103,135 @@ local function get_formspec(data)
 					fgettext("Enable MP") .. "]"
 			end
 		else
-			if mod.enabled then
-				retval = retval .. "checkbox[5.5,-0.125;cb_mod_enable;" ..
-					fgettext("enabled") .. ";true]"
-			else
-				retval = retval .. "checkbox[5.5,-0.125;cb_mod_enable;" ..
-					fgettext("enabled") .. ";false]"
-			end
+			retval = retval .. "checkbox[5.5,-0.125;cb_mod_enable;" ..
+				fgettext("enabled") .. ";" .. tostring(mod.enabled) .. "]"
 		end
 	end
 	if enabled_all then
 		retval = retval ..
-			"button[8.75,0.125;2.5,0.5;btn_disable_all_mods;" .. fgettext("Disable all") .. "]"
+			"button[8.75,0.125;2.5,0.5;btn_disable_all_mods;" ..
+			fgettext("Disable all") .. "]"
 	else
 		retval = retval ..
-			"button[8.75,0.125;2.5,0.5;btn_enable_all_mods;" .. fgettext("Enable all") .. "]"
+			"button[8.75,0.125;2.5,0.5;btn_enable_all_mods;" ..
+			fgettext("Enable all") .. "]"
 	end
-	retval = retval ..
+	return retval ..
 		"tablecolumns[color;tree;text]" ..
-		"table[5.5,0.75;5.75,6;world_config_modlist;"
-	retval = retval .. modmgr.render_modlist(data.list)
-	retval = retval .. ";" .. data.selected_mod .."]"
+		"table[5.5,0.75;5.75,6;world_config_modlist;" ..
+		modmgr.render_modlist(data.list) ..
+		";" .. data.selected_mod .."]"
+end
 
-	return retval
+-- makes a list of mod ids indexed by their names
+local mod_ids
+local function get_mod_ids(list)
```

Comment:
`update_mod_ids`, as this doesn't return a value

Reply:
This function is no longer used.